HomeWork/HomeWork/Selenium WD/Functional/Search.cs

```diff
@@ -0,0 +1,33 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
+
+namespace HomeWork.Selenium_WD.Functional
+{
+    internal class SearchField
+    {
+        private IWebDriver driver;
+
+        public SearchField(IWebDriver driver)
+        {
+            this.driver = driver;
+        }
+
+        public void SearchFieldProductInput(string productSearch)
```

Comment:
Please, divide this method for 2 another by logic, first one is performing a search, the second one is verifying results